database/migrations/2022_09_26_123543_add_website_article_reactions_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('website_article_reactions', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedInteger('user_id');
+            $table->unsignedInteger('article_id');
+            $table->string('reaction', 50);
+            $table->boolean('active')->default(true);
```

Comment:
```suggestion
            $table->boolean('active')->default(true);
            $table->foreign('article_id')->references('id')->on('website_articles')->cascadeOnDelete();
```

I think it'd be a great idea to reference the related article to make sure data integrity stays on point 👍🏻